client/my-sites/posts/post-share.jsx

```diff
@@ -71,7 +71,7 @@ const PostSharing = React.createClass( {
 					'is-active': ( this.state.skipped.indexOf( connection.keyring_connection_ID ) === -1 )
 				} ) }
 			>
-				<SocialLogo icon={ connection.service }/>
+				<SocialLogo icon={ connection.service === 'google_plus' ? 'google-plus' : connection.service }/>
```

Comment:
What do you think about using [kebabCase](https://lodash.com/docs/4.17.2#kebabCase) from lodash in here? :)